printing/paper

```diff
@@ -43,36 +43,37 @@ def view(args):
         print(bold(red("Group accounts can't print. Sorry!")))
 
 
-def refund(args):
-    refund = Refund(
+def adjust(args, type):
+    payload = Payload(
         user=args.user,
         time=datetime.now(),
         pages=args.pages,
+        action=type,
```

Comment:
You can just leave `refund` as a method with 1 argument, but use `args.command` for the `action` parameter instead of adding parameter `type` (This is probably what @dkess says and in this case my comment can serve as a clarification).